ToDoList-frontend/src/app/bucket/bucket.component.ts

```diff
@@ -251,6 +297,7 @@ export class BucketComponent implements OnInit {
   popupNewBucketTaskForm() {
     this.showNewBucketTaskForm = !this.showNewBucketTaskForm;
     this.initializeNewBucketTaskForm();
+    //this.addNewBucketTaskFormGroup.patchValue({ assigneeControl: this.selectAssignee() });
```

Comment:
Z reguly zostawianie zakomentowanego kodu tylko dlatego, ze moze sie przydac w przyszlosci nie ma sensu. Jak bedzie potrzeba, to sie doda ;)

Reply:
Masz rację, to samo bym powiedział o console.log zbędnych. Trochę wyczyściłem